autoware_adapi_v1_msgs/system/msg/MrmState.msg

```diff
@@ -13,6 +13,7 @@ uint16 MRM_FAILED = 4
 uint16 NONE = 1
 uint16 EMERGENCY_STOP = 2
 uint16 COMFORTABLE_STOP = 3
+uint16 PULL_OVER = 3
```

Comment:
@asana17 
`COMFORTABLE_STOP` and `PULL_OVER` have the same value.
Is this correct?

Reply:
```suggestion
uint16 PULL_OVER = 4
```

Reply:
I made the mistake. Thank you for the fix.